web/requirements-frozen.txt

```diff
@@ -0,0 +1,61 @@
+asn1crypto==0.24.0
+atomicwrites==1.1.5
+attrs==18.1.0
+certifi==2018.4.16
+cffi==1.11.5
+chardet==3.0.4
+click==6.7
+coreapi==2.3.3
+coreschema==0.0.4
+cryptography==2.1.4
+datapunt-authorization-django==0.2.20
+datapunt-authorization-levels==0.1.3
+Django==2.0.7
+django-cors-headers==2.3.0
+django-debug-toolbar==1.7
+django-extensions==2.0.7
+django-filter==1.1.0
+djangorestframework==3.8.2
+djangorestframework-gis==0.13
+git+https://github.com/Amsterdam/drf_amsterdam.git#egg=drf_amsterdam
+drf-extensions==0.3.1
+drf-yasg==1.9.1
+flake8==3.5.0
+flex==6.13.2
+future==0.16.0
+idna==2.7
+inflection==0.3.1
+itypes==1.1.0
+Jinja2==2.10
+jsonpointer==1.14
+jsonschema==2.6.0
+lxml==4.2.3
+Markdown==2.6.11
+MarkupSafe==1.0
+mccabe==0.6.1
+more-itertools==4.2.0
+openapi-codec==1.3.2
+pluggy==0.6.0
+psycopg2-binary==2.7.5
+py==1.5.4
+pycodestyle==2.3.1
+pycparser==2.18
+pyflakes==1.6.0
+PyJWT==1.6.4
+pytest==3.6.2
+pytest-django==3.3.2
+pytest-flake8==1.0.1
+python-dateutil==2.7.3
+pytz==2018.5
+PyYAML==3.12
+requests==2.19.1
+rfc3987==1.3.7
+ruamel.yaml==0.15.42
+six==1.11.0
+sqlparse==0.2.4
+strict-rfc3339==0.7
+swagger-spec-validator==2.3.1
+uritemplate==3.0.0
+urllib3==1.23
+uWSGI==2.0.17
+validate-email==1.3
```

Comment:
These are extremely many requirements, but anyway if it is pip freeze --local or something like that then it is what it is....

Reply:
I checked by comparing output of 
`pip freeze --local`
and
`pip freeze`
they both yield the same results when run in the docker container of the web service.